site/docs/v1/tech/tutorials/switch-search-engines.adoc

```diff
@@ -0,0 +1,89 @@
+---
+
+As of version 1.16, FusionAuth supports two search engines:
+
+* database
+* elasticsearch
+
+and you can easily switch between them. The strengths and limitations of each search engine type link:/docs/v1/tech/core-concepts/users#user-search[are documented here].
+
+This tutorial walks you through changing your search engine. Reasons for modifying it vary; you may be simplifying your deployment strategy by using the database search engine. Or perhaps you need the advanced searching options of Elasticsearch. Either way, it's an easy switch.
+
+=== Prerequisites
+
+This tutorial assumes you have both the application and the search services running on Linux. You also must be running at least version 1.16.0. 
+
+If you don't, head on over to the link:/docs/v1/tech/installation-guide/fusionauth-search[installation guide]. If you already have the application running, but need to install the search service, link:/docs/v1/tech/installation-guide/fusionauth-search[here are instructions].
+
+If you are running FusionAuth via Docker, a Windows service, or some other mechanism, you'll have to stop and start FusionAuth in an installation specific manner.
+
+=== From Elasticsearch to Database
+
+Stop FusionAuth:
+
+```
+sudo service fusionauth-app stop
+sudo service fusionauth-search stop
+```
+
+Locate your FusionAuth configuration file, it might be at a location similar to `/usr/local/fusionauth/config/fusionauth.properties`.
+
+Comment out or remove the `fusionauth-search.*` properties and the `fusionauth-app.search-servers` value. 
+
+Add or update the `fusionauth-app.search-engine-type` key to have the value `database`.
+```
+fusionauth-app.search-engine-type=database
+```
+
+Restart the FusionAuth application:
+
+```
+sudo service fusionauth-app start
+```
+
+Log into FusionAuth. Navigate to [breadcrumb]#User# and search for a user by email address or first name to confirm that the search engine is working. 
+
+image::tutorials/database-user-search.png[User search with the database search engine,width=1200,role=shadowed]
+
+=== From Database to Elasticsearch 
+
+
+Stop FusionAuth:
+
+```
+sudo service fusionauth-app stop
+```
+
+Locate your config file, it might be at a location similar to `/usr/local/fusionauth/config/fusionauth.properties`.
+
+Uncomment or add the `fusionauth-app.search-servers` value. This must point to your Elasticsearch servers. 
+
+```
+fusionauth-app.search-servers=http://localhost:9021
+```
+
+Add or uncomment and update additional `fusionauth-search.*` properties, as documented in the link:../reference/configuration[configuration reference]. In addition, add or update the `fusionauth-app.search-engine-type` key to have the value `elasticsearch`.
+
+```
+fusionauth-app.search-engine-type=elasticsearch
+```
+
+Restart the FusionAuth application, making sure to start the `fusionauth-search` service as well.
+
+```
+sudo service fusionauth-search start
```

Comment:
Similar to above, this assume we are using a local Elasticsearch service, and it assumes our service. It could be external, docker etc. We could optionally state up front that these steps can be skipped if you're running Elasticsearch somewhere else or via Docker etc. Or say it each time we have the command for start and stop.